src/components/Trade/hooks/useTradeRoutes/useTradeRoutes.ts

```diff
@@ -18,13 +23,26 @@ export const useTradeRoutes = (): {
   const buyAsset = getValues('buyAsset')
   const sellAsset = getValues('sellAsset')
   const assets = useSelector(selectAssets)
-  const feeAsset = assets['eip155:1/slip44:60']
+  const feeAsset = assets[ETHEREUM_CAIP19]
 
   const setDefaultAssets = useCallback(async () => {
     try {
       const [sellAssetId, buyAssetId] = getDefaultPair()
       const sellAsset = assets[sellAssetId]
-      const buyAsset = assets[buyAssetId]
+      let isEthCaip19
+      let isEthereumChain
+
+      if (defaultBuyAssetId) {
```

Comment:
No need to define these two variables as let, then assign them inside the `defaultBuyAssetId` truthiness check condition. That's just comparing two string primitives, which has virtually no cost.